lib/compat/wordpress-6.3/get-global-styles-and-settings.php

```diff
@@ -124,3 +124,35 @@ function wp_get_block_css_selector( $block_type, $target = 'root', $fallback = f
 function gutenberg_get_remote_theme_patterns() {
 	return WP_Theme_JSON_Resolver_Gutenberg::get_theme_data( array(), array( 'with_supports' => false ) )->get_patterns();
 }
+
+/**
+ * Gets the styles resulting of merging core, theme, and user data.
+ *
+ * @since 6.3.0
```

Comment:
```suggestion
 * @since 5.9.0
```

Reply:
We can add a new line saying:

```php
@since 6.3.0 the internal link format "var:preset|color|secondary" is resolved to "var(--wp--preset--font-size--small)" so consumers don't have to.
```